### DISCUSSION TITLE
Non-standard call signs are not entered into the log

### DISCUSSION THREAD

**Discussion by MikleBas** [magicbug/Cloudlog#3123] [Q&A] [answered]
Today I made a QSO, both with non-standard call signs
1. **4U29MAY**
2. **D1IY**
And both times, when I pressed the Save button, nothing happened, is it just me (and this is a local glitch of my server) or someone try entering these call signs into the communication window, will they be recorded in the log?
Thank you

PS magazine version v2.6.0
all communications were carried out in CW mode

**Comment by magicbug** [comment]
If you update Country File 1. should go in without a problem else manually select the dxcc.

2. isn't a valid callsign.

**Reply by MikleBas** [reply, marked as answer]
1. Updated the files, it helped (although the crown requires auto-update)
Thank you!

**Reply by MikleBas** [reply]
p. 2 The time will come for these regions of Russia to be recognized, or not, but this is already politics, it has no place in Radio yet.. ;)

**Reply by magicbug** [reply]
I'm not refering to politics its a prefix that hasn't been issued by the ITU simple as that.

**Reply by MikleBas** [reply]
Ok!

**Reply by R1BLH** [reply]
> p. 2 The time will come for these regions of Russia to be recognized, or not, but this is already politics, it has no place in Radio yet.. ;)

Previous versions of the Cloudlog were able to log QSOs with such callsigns without problems:
  Digi mode QSO, last logged on 08.04.2024 in my log via the Gridtracker.
  SSB mode QSO, last logged on 11/04/2022 in my log manually.

Now (v.2.6.13) manual logging is blocked. API logging need to be checked.

![изображение](https://github.com/magicbug/Cloudlog/assets/16667173/16c13f92-e637-4d8f-be21-693728ac3dc5)

**Reply by MikleBas** [reply]
Привет!
Через гридтреккер в цифре Ок, а вот сегодня в ручном режиме (CW) оба позывных не вносились в лог..
by ub3ayq

**Reply by R1BLH** [reply]
Ну да, я тоже проверил сегодня. При записи вручную работает фильтр по DXCC. Питер выше писал про это.
Через API запись работает, только что проверил.

Yes, I checked it today. When loggng manually, the DXCC filter works. Peter wrote about this above.
Recording via API works, I just checked it.

**Reply by R1BLH** [reply]
> I'm not refering to politics its a prefix that hasn't been issued by the ITU simple as that.

Is it possible to add this prefix manually to the local database? Or create a special table for overriding ITU data? It would also be very good for localization purposes -- to translate the country names, for example.
